docs/Microservices/RPS/securityRPS.md

```diff
@@ -32,20 +33,23 @@ This is the password that is used to encrypt the provisioning certificate .pfx f
 RPS fetches the password from Vault and will use it when it is needed to decrypt a provisioning certificate.
 
 ### 5 Device configuration information sent to Intel® AMT device
-This data is a set of information that Intel® AMT firmware will use to establish trust and then activate the Intel® AMT device.  Contained in this information is the hashed remote admin password and the MEBX password.  It is important to protect this set of information while it is being used by RPS and while in transit to the Intel® AMT device.  Ensuring that a secure (TLS encrypted) WebSocket connection is used when RPS is communicating with the client device will protect this data while in transit.  This set of information uses nonces to prevent replay of this data.
+This data is a set of information that Intel® AMT firmware will use to establish trust and then activate the Intel® AMT device.  Contained in this information is the hashed remote admin password and the MEBX password.  It is important to protect this set of information while it is being used by RPS and while in transit to the Intel® AMT device.  Ensuring that a secure (TLS encrypted) WebSocket connection is used when RPS is communicating with the client device will protect this data while in transit.  This set of information utilizes nonces to prevent replay of this data.
```

Comment:
"Intel® AMT firmware uses configuration information to establish trust and then activate the Intel® AMT device. The information contains the hashed remote admin password and MEBX password. Protect this information while it is being used by RPS and while in transit to the Intel AMT device. Ensure that a secure (TLS-encrypted) WebSocket is used when RPS is communicating with the client device. This will protect data in transit.  The configuration information uses nonces to prevent replay of this data.

Reply:
updated!